Cargo.toml

```diff
@@ -15,7 +15,10 @@ fxhash = "0.2"
 log = "0.4"
 num-traits = "0.2"
 spirv = { package = "spirv_headers", version = "1" }
-glsl = "4.0.3"
+glsl = { version = "4.0.3", optional = true }
```

Comment:
does it have to be "4.0.3" specifically? Unless you require a feature that was introduced in this exact version, let's just put "4" in here